src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -185,17 +185,17 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 		}
 
 		contextArr
-			.push(...this.instructionAttachmentsPart.chatAttachments);
+			.push(...this.promptInstructionsAttachmentsPart.chatAttachments);
 
 		return contextArr;
 	}
 
 	/**
-	 * Check if the chat input part has any prompt instruction attachments.
+	 * Check if the chat input part has any prompt file attachments.
 	 */
-	public get hasInstructionAttachments(): boolean {
+	public get hasPromptFileAttachments(): boolean {
 		// if prompt attached explicitly as a "prompt" attachment
-		if (this.instructionAttachmentsPart.empty === false) {
+		if (this.promptInstructionsAttachmentsPart.hasPromptFile) {
```

Comment:
This breaks the "run" editor action - we now rely on the "prompt" language ID to enable the action, hence file with any extension can now be run as a prompt, but this check conditions on the `.prompt.md` extension only.